app/main.py

```
@@ -0,0 +1,49 @@
+
+
+# Startup event to create database tables
+@app.on_event("startup")
+async def on_startup():
+    async with engine.begin() as conn:
+        await conn.run_sync(models.Base.metadata.create_all)
+
+
+# Get all ToDo items
+@app.get("/todos", response_model=list[schemas.ToDoResponse])
+async def read_todos(db: AsyncSession = Depends(get_db)):
```

Ruff (0.8.2)

Missing return type annotation for public function `read_todos`

(ANN201)